rebar.config.script

```diff
@@ -20,13 +20,34 @@ FilterDepsRocksDb = fun(Deps) ->
                         end
                     end,
 
+OverlayReplacements = [
+                       "bin/epoch",
+                       "bin/extensions/check_config",
+                       "bin/extensions/keys_gen",
+                       "bin/extensions/peer_key",
+                       "bin/extensions/messages_hash",
+                       "bin/extensions/export_chain"
+                      ],
```

Comment:
This list seems prone to get out of sync. Would it be possible, at build time, to cause error if any of the following constraints is not adhered to:
* For each item X of the `OverlayReplacements`, there must be a `{copy, Src0, Dest0}` with `Dest0` equal to X.
* Each item among the `{copy, Src0, Dest0}` items where `Dest0` starts with `"bin/extensions/` must be such that `Dest0` is among `OverlayReplacements`.

Reply:
made generic